perl/bin/alleleCounterToJson.pl

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/perl
+
+##########LICENCE##########
+# Copyright (c) 2014-2021 Genome Research Ltd.
+#
+# Author: CASM/Cancer IT <cgphelp@sanger.ac.uk>
+#
+# This file is part of alleleCount.
+#
+# alleleCount is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation; either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+##########LICENCE##########
+
+use strict;
+use Carp;
+use English qw( -no_match_vars );
+use warnings FATAL => 'all';
+
+use Getopt::Long 'GetOptions';
+use Pod::Usage;
+
+use Sanger::CGP::AlleleCount;
+use Sanger::CGP::AlleleCount::ToJson;
+
+{
+  my $options = option_builder();
+  $options->{'o'} = '/dev/stdout' unless(defined $options->{'o'});
+  run($options);
+}
+
+sub run {
+  my ($options) = @_;
+  my $json_string = Sanger::CGP::AlleleCount::ToJson::alleleCountToJson($options->{'a'}, $options->{'l'});
+  my $OUT;
+  open($OUT, '>', $options->{'o'}) or croak("Error opening file for output: $!");
+    print $OUT "$json_string";
+  close($OUT) or croak("Error closing output file for JSON conversion: $!");
```

Comment:
Does this work without throwing error about attempting to open/close stdout?

I thought you needed to do:

```perl
my $OUT = *STDOUT;
if(defined $options->{'o'}) {
    open($OUT, '>', $options->{'o'}) or croak("Error opening file for output: $!");
}

print $OUT "$json_string";

if(defined $options->{'o'}) {
    close($OUT) or croak("Error closing output file for JSON conversion: $!");
}
```

Reply:
No errors thrown in testing, but added the catch in case.